sdks/wasm/print-table.py

```diff
@@ -6,7 +6,7 @@
 
 import sys
 
-prefix=" (elem (get_global $tableBase) "
+prefix=" (elem (i32.const 1) "
```

Comment:
What's this?

Reply:
Its a tool to print out the indirect function table from a wasm text file so if you print out a function pointer in C code, you can find the corresponding wasm function.

Reply:
I just meant specifically that it wasn't clear why you were replacing a get_global with a constant, since it didn't look like you changed the global. Mostly curious

Reply:
The wasm generated by the recent emscripten is different for the table:
(elem (i32.const 1) $mono_wasm_invoke_js $mono_wasm_invoke_js_marshalled...